pro/src/screens/IndividualOffer/DetailsScreen/DetailsEanSearch/DetailsEanSearch.tsx

```diff
@@ -105,10 +117,17 @@ export const DetailsEanSearch = ({
     }
   }
 
+  const onEanClear = () => {
+    resetForm()
+    setWasCleared(true)
```

Comment:
(question) est-ce qu'il y a vraiment besoin du `state` ?
Cette ligne ne pourrait pas être remplacée par ça ? (et supprimer le useEffect)
```suggestion
    if (inputRef.current) {
      inputRef.current.focus()
    }
```